File: examples/single-project-multiple-test-sets/src/test/java/org/example/test/ExampleTest.java
package org.example.test;

import org.example.ExampleLib;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertTrue;

public class ExampleTest {

    @Test
    public void testA() {
        assertTrue(new ExampleLib().run());
    }
}

File: examples/single-project-kotlin/src/test/kotlin/org/example/test/ExampleTest.kt
package org.example.test

import com.code_intelligence.jazzer.junit.FuzzTest
import org.example.ExampleLib

import org.junit.jupiter.api.Assertions.assertTrue

class ExampleTest {

    @FuzzTest
    fun fuzzTest(data: ByteArray) {
        assertTrue(ExampleLib().run());
    }
}
